tutorials/blender_procedural_datasets.md

```diff
@@ -0,0 +1,109 @@
+generation of procedural datasets (examples shown below). This script has the
+following features:
+
+1. Generation of SDF models compatible with Ignition Fuel.
+
+2. Exporting visual and collision mesh geometry (at different levels of detail).
+
+3. Linking to existing PBR textures via symlink or hard copy.
+
+4. Estimation of inertial properties assuming uniform density.
+
+5. Randomization of parameters, e.g. mass and surface friction.
+
+## Procedural Mesh Geometry
+
+[url_blender_docs_geometry_nodes]: https://docs.blender.org/manual/en/latest/modeling/geometry_nodes/introduction.html
+[rock_blend_file]: https://github.com/ignitionrobotics/ign-gazebo/tree/ign-gazebo6/tutorials/files/blender_procedural_generation/rock.blend
+[Geometry Nodes][url_blender_docs_geometry_nodes] is a feature of Blender that
+enables the creation and modification of mesh geometry through node-based
+pipelines. By combining procedural operations with randomness, a seemingly
+infinite number of model variations can be achieved by changing a seed of the
+pseudo-random generator. For example, below is a showcase of randomly generated
+geometry for rocks. The corresponding `.blend` file can be accessed here (tested
+with Blender 3.0/3.1): [rock.blend][rock_blend_file]. This file can be used as a
+template for the creation of other custom datasets.
+
+@image html https://github.com/ignitionrobotics/ign-gazebo/tree/ign-gazebo6/tutorials/files/blender_procedural_generation/example_rock.gif "Procedural Generation Example" width=100%
+
+[url_blender_tutorial_geometry_nodes_0]: https://youtube.com/watch?v=4WAxMI1QJMQ&list=PLjEaoINr3zgFX8ZsChQVQsuDSjEqdWMAD&index=9
+[url_blender_tutorial_geometry_nodes_1]: https://youtube.com/watch?v=XSkaM-8Vgz8
+Although Geometry Nodes enable relatively simple and fast creation of datasets,
+it can be tricky to get started. Therefore, consider following one of the online
+tutorials if you are interested in creating your own datasets. As an example,
+[Geometry Nodes Tutorial (Donut part 9)][url_blender_tutorial_geometry_nodes_0]
+provides an introduction to Geometry Nodes (consider watching the entire series
+to get familiar with Blender). Similarly, more complex tutorials are available
+such as [Plant Growth with Fields][url_blender_tutorial_geometry_nodes_1].
+
+<!-- ## Procedural Textures and Materials -->
+<!-- TODO[feature]: Document procedural materials once implemented -->
+
+## Estimation of Inertial Properties
+
+[url_trimesh_org]: https://trimsh.org
+Script [`procedural_dataset_generator.py`][procedural_dataset_generator_script]
+also supports the automatic estimation of inertial properties by the use of the
+[trimesh][url_trimesh_org] Python module. During the first execution of the
+script, this module is automatically installed inside the Python environment of
+the Blender. Note that this requires permissions to write into Blender
+installation directory. Mass, inertia and centre of mass are estimated for each
```

Comment:
The user will receive an error if the Python module cannot be installed (either due to lack of permissions or because of read-only directory in the case of `snap` installation). The user is then instructed in the error message to either install it manually or use the `--python-use-system-env` option. Alternatively, they are instructed manually disable the estimation of the inertial properties (either by changing the default parameter in the script or via a CLI argument).

I improved a the tutorial section about under `## Known Issues`.